resources/lib/sosac.py

```diff
@@ -196,23 +206,22 @@ def list(self, url):
 
         return [self.dir_item(title="I failed", url="fail")]
 
+    def list_by_genres(self, url):
+        if "?" + GENRE_PARAM in url:
+            return self.list_xml_letter(url)
+        else:
+            result = []
+            page = util.request(url)
+            data = util.substr(page, '<select name=\"zanr\">', '</select')
```

Comment:
Není můj kód, viz https://github.com/kodi-czsk/plugin.video.sosac.ph/pull/44
Pull jsem dělal na jeho úpravách protože fixuje jiné bugy (co mě štvaly=napr. nemůžu vytvořit adresář...) a přidává novou funkčnost žádanou ostatními. Kód jsem nestudoval, zvlášť když nejsem programátor:)